README.md

```diff
@@ -1 +1,35 @@
-# My-Capstone-Project
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> HTML/CSS & JavaScript capstone project - Conference page.
+
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+
+## Live Demo
+
+[Live Demo Link](https://gasangw.github.io/My-Capstone-Project/)
+
+
+## Authors
+
+👤 **Gasangwa Thomas**
+
+- GitHub: [@githubhandle](https://github.com/gasangw)
+- Twitter: [@twitterhandle](https://twitter.com/ThomasGasangwa)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/gasangw/My-Capstone-Project/issues).
+
```

Comment:
- [ ] Give credits for others' contributions is always a nice way to say thanks, please be sure to add an "Acknowledgements" section and give appropriate credits to Cindy Shin, the author of this UI design